website/pages/docs/deployment.md

```diff
@@ -125,6 +125,77 @@ Then run the command `now` in your project, you will be given a `now.sh` URL as
 
 Check out [Now for GitHub](https://zeit.co/docs/v2/integrations/now-for-github) and [Now for GitLab](https://zeit.co/docs/v2/integrations/now-for-gitlab/) for continuous intergration.
 
+## [Firebase](https://firebase.google.com/)
+
+To host and deploy your site to Firebase.
+
+### Step 1: Log into Firebase and create a new project or choose from an existing one
+
+### Step 2: Create a new web app from the project dashboard and select "Also setup Firebase Hosting for this app"
+
+### Step 3: Add the required Firebase scripts to the body of your site by editing a file called `saber-browswer.json` at the root of your project to include the following.
```

Comment:
this is probably unnecessary.

Reply:
Sure, whatever you feel is appropriate. Depends how much hand-holding you want to give.